WordPress/Classes/ViewRelated/Stats/Today Widgets/Shared Views/WidgetUnconfiguredCell.swift

```diff
@@ -45,10 +55,13 @@ private extension WidgetUnconfiguredCell {
                 return LocalizedText.configureAllTime
             case .thisWeek:
                 return LocalizedText.configureThisWeek
+            case .loadingFailed:
+                return LocalizedText.loadingFailed
```

Comment:
Oooh this is nice! I like this naming convention. I've always done `Constants.loadingFailed`